tests/python/driver/tvmc/test_autoscheduler.py

```diff
@@ -26,29 +24,32 @@
 
 
 def _get_tasks(model):
-    mod, params = tvmc.frontends.load_model(model)
-    tasks, weights = tvmc.autotuner.autoscheduler_get_tuning_tasks(mod, params, "llvm")
+    tvmc_model = tvmc.frontends.load_model(model)
+    tasks, weights = tvmc.autotuner.autoscheduler_get_tuning_tasks(
+        tvmc_model.mod, tvmc_model.params, "llvm"
+    )
     return (tasks, weights)
 
 
-def _autoscheduler_test_helper(
-    model, tmpdir_name, tasks_weights=None, early_stopping=1, tuning_records=None
-):
-    tasks, weights = tasks_weights if tasks_weights else _get_tasks(model)
+def _autoscheduler_test_helper(model, tmpdir_name, early_stopping=1, tuning_records=None):
+    tvmc_model = tvmc.frontends.load_model(model)
+    tvmc_model.tuning_records = tuning_records
```

Comment:
another strange instance where we modify an attribute of TVMCModel

Reply:
I've done some refactoring so that the input objects arent modified in any way.